### DISCUSSION TITLE
Preparing for `v3.0.0`

### DISCUSSION THREAD

**Discussion by abhiabhi94** [Radi85/Comment#239] [Ideas]
Hey @Radi85, I was looking to work on #179 in the coming weeks. I think the database structure would work, but we would need to tinker the ORM and the view structure, and also the user interface. Do you want it to wait?

If not, I can send a patch updating the `changelog` for the upcoming release. Maybe we can release a new-version and fix some bugs, and parallel now, start to work on the major version bump?

If you have something else in mind, feel free to share it.

**Comment by Radi85** [comment]
> Hey @Radi85, I was looking to work on #179 in the coming weeks. I think the database structure would work, but we would need to tinker the ORM and the view structure, and also the user interface. Do you want it to wait?

Hey @abhiabhi94, thanks a lot for your effort.
I think we should firstly implement the features (opened issues) specified for the versions 2.8 and 2.9 then we can bump the version with this main feature #179 and less side features and bug fixes.

> If not, I can send a patch updating the `changelog` for the upcoming release. Maybe we can release a new-version and fix some bugs, and parallel now, start to work on the major version bump?

However, we may start with beta release for version `3.0.0` and make the release after 2.9 is released.

> If you have something else in mind, feel free to share it.

At the moment I am so busy and cannot have much thought about DAB therefore, I am waiting for other contributors to take care of the opened issues and share their ideas.

**Reply by abhiabhi94** [reply]
Thank you for the response.

> I think we should firstly implement the features (opened issues) specified for the versions 2.8 and 2.9 then we can bump the version with this main feature #179 and less side features and bug fixes.

Planned issues for `2.8` include adding support for adding images and removing bootstraps. I don't know whether I have enough motivation to play around with that amount of CSS anymore 😆 .

> At the moment I am so busy and cannot have much thought about DAB therefore, I am waiting for other contributors to take care of the opened issues and share their ideas.

No issues, thank you for your reviews. Just a confirmation, would it be okay with you if I make the releases in the mean time? Just asking because you said earlier that you wanted to handle the releases.

If yes, is there anything specific that you may want me to be aware of when making the releases?
